README.md

```diff
@@ -130,6 +131,9 @@ There isn't a current roadmap for this plugin, but it is in active development.
 
 If you have a feature request, please add it as an [issue](https://github.com/intelsdi-x/snap-plugin-publisher-graphite/issues/new) and/or submit a [pull request](https://github.com/intelsdi-x/snap-plugin-publisher-graphite/pulls).
 
+### Known issues
+Graphite plugin does not check whether on address to which it publishes griphite is running or not. If exists any service on specific address and port Graphite Publisher should not crash.
```

Comment:
I'm not sure I understand the issue being described. Does the following description accurately reword what you are saying? 
```
Currently, this plugin does not perform a check on the publishing address to see if graphite is running or not. If a service is already using that port or address, the plugin should not crash.  
```
@Armatorix, if I understand correctly, this could be broken down to two issues. 1: perform check. 2: ensure plugin doesn't crash. 

Please open issues in github to document these issues.